department-of-veterans-affairs/va.gov-cms
# VACMS-2489: Related Information

…tent type, added related links teaser on QA content type, created new related benefits entity browser.

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/2489

## Testing done


## Screenshots


## QA steps

As an editor
1. Login
2. Confirm the following AC:

 - [x] "VA Benefit Hubs" and "Related links" fields should be added to the following content types: Q&A, FAQ - multiple Q&As, Checklist, Step-by-Step, Media list - Images, Media list - Videos.

VA Benefit Hubs field
 - [x] entity reference field in which only "Benefits Hub Landing Page" content type can be referenced. Form widget "Entity Browser - Table". New view should be created instead of using the existing /admin/structure/views/view/benefits_hub_list, since Entity Browser field widget will display the full title of the hub including VA prefix and we need parity between selection and display UI.
- Spec: field_benefit_categories | label "VA Benefit Hubs" | cardinality - max 3
 - [x] referenced Hubs can be reordered via drag and drop
 - [x] default Hub is always "VA health care"

Related links field
 - [x] entity reference revisions field to "Link teaser" paragraph, to allow for addition of internal/external links. Spec: field_related_links | label "Related links" | cardinality - multiple, max 5

Form display
 - [x] Both new fields are displayed at the bottom of the form above Editorial Workflow field group. "VA Benefit Hubs" fields is displayed above "Related links" field. No field group needed around new fields.

other ACs
 - [x] node display settings are configured for each content type: ensure that fields are displayed in correct order. Both new fields are at the bottom of the page.
- [x] spec tool is updated

## Definition of Done
